tests/03-auth.spec.ts

```diff
@@ -1,9 +1,10 @@
 import { chai } from 'mochainon';
-import { ResinAuth } from '../lib/auth';
 import { TokenType } from '../lib/token';
 import apiKeyFixtures from './fixtures/api-keys';
 import jwtFixtures from './fixtures/jwts';
 
+const ResinAuth = require('../lib/auth');
```

Comment:
We do need to change the import, you're right, but not quite like this. TypeScript only knows about import statements that look like one of:

```ts
import X from 'module';
import X = require('module');
```

when you use `const ResinAuth = require('../lib/auth')`, you're writing code that will work at runtime, but which TypeScript isn't aware of (simply because it's using `require` as a normal function in an expression, instead of a special top-level statement using `import`), so ResinAuth won't be typed correctly.

More practically: if you do this tslint will fail the build (as you can see on travis for this PR).

Instead, this should probably be something like:

```ts
import * as ResinAuth from '../lib/auth';
```

that does what you're expecting, but also works nicely with TypeScript.

Reply:
Might actually need to be `import ResinAuth = require('../lib/auth')` - either one is fine, whatever works.